astropy/utils/iers/iers.py

```diff
@@ -191,7 +191,7 @@ class Conf(_config.ConfigNamespace):
         ["error", "warn", "ignore"],
         "IERS behavior if the range of available IERS data does not "
         "cover the times when converting time scales, potentially leading "
-        "to degraded accuracy.",
+        "to degraded accuracy.  Applies only to when using only IERS-B data.",
```

Comment:
Addition should perhaps read " Applies only when using IERS-B data on its own."

Reply:
Sure, reads better